orchestration/src/main/java/com/sap/ai/sdk/orchestration/NoTypeInfoMixin.java

```diff
@@ -0,0 +1,7 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+/** Mixin to suppress @JsonTypeInfo for oneOf interfaces. */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NONE)
+public interface NoTypeInfoMixin {}
```

Comment:
(Question/Minor)

Can this class be package-private?
If not, could we append `@Beta` annotation?